### ISSUE TITLE
Multipage Editor (XML) loosing its tabs.

### ISSUE THREAD

**Issue by ghost** [rogerdudler/eclipse-ui-themes#11]
In the themes available by default, everything's fine. The tabs at the bottom of the editor are visible.

![before applying Dark Juno](http://iontech.files.wordpress.com/2012/08/before2.png)

And once the the Dark Juno theme is applied, those tabs at the bottom of the editor disappear.

![after applying Dark Juno](http://iontech.files.wordpress.com/2012/08/after.png)

**Comment by ahcox** [comment]
I see this too on Ubuntu 12.10 (before it is released), Eclipse 4.2 (Juno).

http://stackoverflow.com/questions/12534818/how-do-i-force-eclipse-to-show-xml-layouts-and-styles-as-text

eclipse dumps this on startup:

```
~$ eclipse
Failed to find defStyle: listViewStyle
Failed to find defStyle: listViewStyle
Failed to find defStyle: listViewStyle
Failed to find defStyle: textViewStyle
Failed to find defStyle: textViewStyle
Failed to find defStyle: textViewStyle
Failed to find defStyle: buttonStyle
Failed to find defStyle: buttonStyle
Failed to find defStyle: buttonStyle
Failed to find defStyle: buttonStyle
Failed to find defStyle: buttonStyle
Failed to find defStyle: buttonStyle
Failed to find defStyle: buttonStyle
Failed to find defStyle: buttonStyle
Failed to find defStyle: buttonStyle
```

**Comment by emilio-simoes** [comment]
I'm having this issue to both on Windows 7 x64 and Kubuntu 12.04 x64 with Eclipse Juno.

**Comment by ghost** [comment]
@ahcox @emilio-simoes Yeah! No idea what @rogerdudler is upto with this.

**Comment by rogerdudler** [comment]
I actually don't know what the problem is on that one. I think it's a general theming problem. I have to get in touch with the core team to fix this.

**Comment by ghost** [comment]
Good luck @rogerdudler. We love your work, and wish to see the result soon. :+1:

**Comment by ahcox** [comment]
@iontech @rogerdudler, Yeah to what iontech said. I am still using the theme and like it very much. I wrote up my steps to getting a fully dark Eclipse Juno on Ubuntu here: http://www.hoogli.com/items/Light_on_Dark_Eclipse_4.2_(Juno)_Themes.html

It is possible to switch tabs with ALT-PgUp & ALT-PgDn even though you can't see the GUI elements, which is faster and saves screen real estate. So this is actually a feature in my book ;-).

I am very happy with the result: http://hoogli.com.nyud.net/img/dark_eclipse_juno_theme_003.png

Thanks Roger!

**Comment by elbowz** [comment]
any news? I have the same problem on Kubuntu 12.10

**Comment by ghost** [comment]
@elbowz Nope. No update. Still waiting. @rogerdudler and his team might be real busy with other stuff. Hope to get a patch soon.

**Comment by rogerdudler** [comment]
I am indeed... If someone of you can help out, I would really appreciate it.

**Comment by ghost** [comment]
@rogerdudler try removing the line: swt-tab-renderer: url(bundleclass://org.eclipse.e4.ui.workbench.renderers.swt/org.eclipse.e4.ui.workbench.renderers.swt.CTabRendering);

from the CTabFolder element

**Comment by ghost** [comment]
you should get something like http://snag.gy/T9TQD.jpg

**Comment by rogerdudler** [comment]
@scela are you able to create a pull request for that?

**Comment by ghost** [comment]
https://github.com/rogerdudler/eclipse-ui-themes/pull/16

also more details will follow on https://bugs.eclipse.org/bugs/show_bug.cgi?id=396780

**Comment by rogerdudler** [comment]
@scela Thanks!!

**Comment by elbowz** [comment]
@scela yeah, now work good! Thanks!

**Comment by ghost** [comment]
Great.

**Comment by DJViking** [comment]
The tabs are again visible, but something else got worse.

The color theme got pitch black. Before it was a grayish black.
Because of this the buttons for dropdownlist, minimize and maximize are invisible. They are there black on black.

How it looks like on a light theme.
![eclipse-screenshot13](https://f.cloud.github.com/assets/3070076/19127/8335c816-48ff-11e2-9d31-bd884889894c.png)

How it now looks with the modified Juno Dark.
![eclipse-screenshot14](https://f.cloud.github.com/assets/3070076/19128/87a37538-48ff-11e2-9555-537051f368f0.png)

How it previosuly looked with Juno Dark.
![eclipse-screenshot15](https://f.cloud.github.com/assets/3070076/19129/8b40d37a-48ff-11e2-80a9-42939d7c39c7.png)

I have no problem with pitch black, but other elements on it cannot be black or they will become invisible.

Edit: Not sure why it becomes pitch black. Any changes I do to the juno.css file, repackage it and it becomes pitch black. Even just changing .MPartStack font-size to 10 from the previous Juno Dark makes it pitch black background.

**Comment by ghost** [comment]
I'm not sure my PR caused this because it has no relation to pitch or no pitch black colors, at least directly.

**Comment by DJViking** [comment]
You're right. It was my attempt to repackage the jar file that was the problem. Repackage created a new MANIFEST file. After repackaging I copied the original MANIFEST file and it worked.
